workers/proto/kitchen_sink/kitchen_sink.proto

```diff
@@ -65,6 +66,7 @@ message DoSignal {
     // Send an arbitrary signal
     HandlerInvocation custom = 2;
   }
+  bool with_start = 3;
```

Comment:
I also considered adding it to `ClientAction` instead. How are we dealing with breaking changes; ie could I move it later or is there a backwards-compatibility guarantee?

Reply:
Breaking changes aren't a problem here